okapi-core/src/main/resources/log4j2.properties

```diff
@@ -1,19 +1,25 @@
 status = error
 name = PropertiesConfig
-packages = org.folio.okapi.common.logging
 
 filters = threshold
 
 filter.threshold.type = ThresholdFilter
 filter.threshold.level = info
 
-appenders = console
+appender.full.type = Console
+appender.full.name = FULL
+appender.full.layout.type = PatternLayout
+appender.full.layout.pattern = %d{HH:mm:ss} [${map:requestid}] [${map:tenantid}] [${map:userid}] [${map:moduleid}] %-5p %-20.20C{1} %m%n
 
-appender.console.type = Console
-appender.console.name = STDOUT
-appender.console.layout.type = PatternLayout
-appender.console.layout.pattern = %d{HH:mm:ss} [$${FolioLoggingContext:requestid}] [$${FolioLoggingContext:tenantid}] [$${FolioLoggingContext:userid}] [$${FolioLoggingContext:moduleid}] %-5p %-20.20C{1} %m%n
+appender.omitted.type = Console
+appender.omitted.name = OMITTED
+appender.omitted.layout.type = PatternLayout
+appender.omitted.layout.pattern = %d{HH:mm:ss} [] [] [] [] %-5p %-20.20C{1} %m%n
 
 rootLogger.level = info
 rootLogger.appenderRefs = info
-rootLogger.appenderRef.stdout.ref = STDOUT
+rootLogger.appenderRef.stdout.ref = OMITTED
+
+logger.full = info, FULL
+logger.full.name = org.folio.okapi.util.ProxyContext
+logger.full.additivity = false
```

Comment:
Having all log levels been set to info is fine.

But I wonder what a sysop needs to change in this log4j2.properties file to reduce the root level to warn but keep the full (ProxyContext) log level at info.

Reply:
I don't know how to do that. Ideas?

Reply:
Using 

    rootLogger.level = warn

works for me. Only the REQ/RES lines logged.

Reply:
I tried `mvn test -Dtest=ProxyTest#testHealth` and it doesn't work but this is not relevant and might be caused by some logging set up in the tests.

When running `java -jar target/okapi-core-fat.jar dev` it works as expected, therefore there's no issue.